README.md

```diff
@@ -296,6 +296,18 @@ Multiplatform **requires the gradle metadata feature**, which you need to enable
 enableFeaturePreview('GRADLE_METADATA')
 ```
 
+### Linking SQLite
+
+By default, SQLDelight links system SQLite into your SQLDelight targets via `-lsqlite3`. If you prefer to link a different SQLite, you can control this behavior using the `linkSqlite` flag in the gradle plugin:
+
+```groovy
+sqldelight {
+  linkSqlite = false
```

Comment:
can you move this documentation to this part of the readme: https://github.com/square/sqldelight/pull/1323/files#diff-04c6e90faac2675aa89e2176d2eec7d8L367 i dont think it warrants a top level section